.github/workflows/fixup.yml

```diff
@@ -0,0 +1,20 @@
+# This workflow is provided via the organization template repository
+#
+# https://github.com/nextcloud/.github
+# https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization
+
+name: Pull request checks
+
+on: pull_request
+
+jobs:
+  commit-message-check:
+    name: Block fixup and squash commits
```

Comment:
doesn't this PR touch a bit too much? I'm okay if it's a bit bigger if you need to combine related changes, but this is not even remotely related to either *newFileMenu* nor *rollup*

Reply:
Yeah, there is also a bump of workflows we should have.
I can split it in multiple PRs